docs/contributing.md

```diff
@@ -0,0 +1,202 @@
+
+\*If you'd like to preview your sidebar changes or if you need to preview how a document containing MDX will build, you'll likely want to check into your branch and run `yarn start` to see the changes. Please reach out to the Experimenter team if you need help.
+
+### GitHub UI
+
+The GitHub user interface allows users to make file changes for those that would prefer not to check into the code repository via CLI. See [the GitHub UI Docs](https://docs.github.com/en/github/managing-files-in-a-repository/editing-files-in-your-repository) if you need further details than what's provided here.
+
+In the following sections, you'll find corresponding links to the GH UI directly under each section title. If you need to edit a file, you'll click on the "Edit" button:
+
+<img src="../img/edit-this-file.png" alt="edit this file" style={{ width: '300px' }} />
+
+You'll be brought to the GitHub editor page for that file. If you're creating a new document, you'll see a similar editor.
+
+When you're satisfied with your changes or new document, at the bottom of the page, enter a commit message and optional description. Be sure to have the option "Create a new branch" selected, the name of which can be anything, but **if you also need to make edits to the sidebar**, you'll want to keep the branch name in mind.
+
+<img src="../img/commit-changes.png" alt="Commit changes" style={{ width: '700px' }} />
+
+Click `Commit changes` and you'll be brought to a pull request view comparing your new branch to `main`. If you don't need to make updates to the sidebar, click `Create pull request`, and you're done!\*
+
+#### Doc Edits + Sidebar Edits
+
+If you also need to make updates to the sidebar, you will follow the GH UI link under "Adding or Modifying Sidebar Links" but replace "main" in the URL with the name of your branch. For example, if the name of your branch is `my-flying-nimbus`, [this would be the link](https://github.com/mozilla/experimenter-docs/edit/my-flying-nimbus/sidebars.js) you'd use to edit that file. Similarly to the instructions above, after you've made the desire changes, enter a commit message and optional description. However, this time, you will select the option **commit directly to the `my-flying-nimbus` branch** instead of creating a new branch.
+
+After committing your changes, you'll be brought back to a pull request view comparing your new branch to `main`. You may need to edit the PR title, but if your files changed look good, click `Create pull request`, and you're done!\*
+
+\*Someone else will need to approve your PR (or give you feedback before approving), and then it will be merged and deployed by GH Pages.
+
+## Create a New Document
+
+> 👉 [Do this in the GitHub UI](https://github.com/mozilla/experimenter-docs/new/main/docs) 👈
```

Comment:
More of a nitpick than anything, but these aren't really quotes. I gather you're using `>` to make it stand out, but maybe we could use an admonition or some other button-y type element?

Reply:
It would also be good to mention/link to [admonitions](https://v2.docusaurus.io/docs/markdown-features/admonitions) somewhere, maybe near the bit about GFM

Reply:
I tried using an admonition here but it honestly seemed to stand out _too_ much. 😆 I get what you mean about it not being a quote though so maybe I should use one just to be proper, like `tip` where I can add a custom title. Good call out on linking that doc.

Reply:
I used the "note" one and it doesn't look too bad! The "info" one didn't seem quite right and the green one, eh. You can let me know what you think if you don't like it. 😛

<img width="879" alt="image" src="https://user-images.githubusercontent.com/13018240/111006845-27e86d00-8353-11eb-946a-56dcacf02d09.png">